README.md

```diff
@@ -2,79 +2,98 @@
+### `development`
 
-### Generate keys for testing
+You can use the `development` command if you're building your own client for the VSP. The command starts the VSP connected to a testing service hosted by the GOV.UK Verify team. The testing service acts as a placeholder for the GOV.UK Verify Hub. This means you can use your local setup to test if your service can respond appropriately to all possible scenarios in a Verify journey.
 
-In order to generate keys for testing, we recommend using [OpenSSL](https://www.openssl.org).
+When running the VSP using the `development` command, it initialises the testing service by:
 
-You can generate a private key by:
-1. generating an RSA key in PEM format
-2. converting the key to base64 encoded PKCS8
+- generating its self-signed keys and certificates
+- adding the keys and certificates to the VSP configuration
+- setting the testing service environment in the VSP configuration
+- initialising a testing session with the testing service
+
+To start the VSP connected to the testing service, run:
 
-Generate an RSA key in PEM format with:
 ```
-openssl genrsa -des3 -passout pass:x 2048 | openssl rsa -passin pass:x -out key-name.pem
+./bin/verify-service-provider development
 ```
 
-Convert the PEM formatted key to base64 encoded PKCS8 for the config file. Print the key to STDOUT with:
+You can use the following command line options to customise the behaviour of the `development` command:
+
+| Option | Description | Default |
+| ------ | ----------- | ------- |
+| `-d MATCHINGDATASET` or <br> `--matchingDataset MATCHINGDATASET`| The identity dataset the testing service will use | [Test identity dataset][identity-dataset] |
+|`-u URL` or<br> `--url URL` | The URL where the testing service will send responses | `http://localhost:8080/SAML2/Response` |
+|`-t TIMEOUT` or<br> `--timeout TIMEOUT` | The timeout in seconds when communicating with the testing service | `5` |
+| `-p PORT` or<br> `--port PORT` | The port the service will use | `50300` |
+|`--host BINDHOST` | The host the service will bind to | `0.0.0.0` |
+
+You can check the application is running by calling the healthcheck endpoint with:
+
 ```
-openssl pkcs8 -topk8 -inform PEM -outform DER -in key-name.pem -nocrypt | openssl base64 -A; echo
+curl localhost:{$PORT}/admin/healthcheck
 ```
 
-### Run
+For more information on building your own client using the `development` command, see the [technical documentation on how to get started][vsp-get-started].
 
-To run the application, export your environment variables and start the application with:
+### `server`
+
+Use the `server` command when running the VSP in an environment containing a full-scale deployment of the GOV.UK Verify Hub, for example the Integration or Production environments.
+
+For example, to run the VSP using the environment and security configuration in `verify-service-provider.yml`, export your environment variables and run:
```

Comment:
Do we need any elaboration here, or are we confident the RP devs will understand they need to look inside the config file to find out which environment variables they need to export?

Reply:
Could link to the 'Configuration' section which talks about environment variables?